Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -25,10 +25,11 @@
 
 {% block linkElement %}
     {% spaceless %}
-        {% set translation_domain = item.extra('label_catalogue', 'messages') %}
         {% if item.extra('on_top') is defined and not item.extra('on_top') %}
+            {% set translation_domain = item.extra('translation_domain') %}
             {% set icon = item.extra('icon')|default(item.level > 1 ? '<i class="fa fa-angle-double-right" aria-hidden="true"></i>' : '') %}
         {% else %}
+            {% set translation_domain = item.extra('label_catalogue') %}
```

Comment:
It is already defaulted here : https://github.com/sonata-project/SonataAdminBundle/pull/4601/files#diff-7aad62c164c47d66f90ce813f6f91c49L53

Anyway it's the default translation domain in symfony `Translator::trans()` method : https://github.com/symfony/translation/blob/master/Translator.php#L192